### ISSUE TITLE
[Issue]: Jellyfin plays particularly quietly

### ISSUE THREAD

**Issue by dadastory** [jellyfin/jellyfin#12036]
### Please describe your bug

The Jellyfin Media Player plays very quietly, the headphones play quietly, the speaker plays very quietly, and the background sound is very loud
![image](https://github.com/jellyfin/jellyfin/assets/84147842/838d67c1-57e9-4602-a202-6a8d9715ad92)

### Reproduction Steps

while play movie

### Jellyfin Version

10.9.0

### if other:

10.9.3

### Environment

```markdown
- OS:
- Linux Kernel:
- Virtualization:
- Clients:
- Browser:
- FFmpeg Version:
- Playback Method:
- Hardware Acceleration:
- GPU Model:
- Plugins:
- Reverse Proxy:
- Base URL:
- Networking:
- Storage:
```

### Jellyfin logs

```shell
no
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by gnattu** [comment]
Can you go to settings -> playback and change the Maximum Allowed Audio Channels to stereo, scroll down, click save, and then try again?

**Comment by dadastory** [comment]
> Can you go to settings -> playback and change the Maximum Allowed Audio Channels to stereo, scroll down, click save, and then try again?

thanks，I have tried but it seem did not have any different

**Comment by gnattu** [comment]
Can you show a screenshot of playback info after you changed that?

**Comment by dadastory** [comment]
> Can you show a screenshot of playback info after you changed that?

![image](https://github.com/jellyfin/jellyfin/assets/84147842/ad757b9f-d52e-4e4e-9a12-bd4ced3bb4b6)

**Comment by gnattu** [comment]
For JMP, I think you need to go to the client settings and change audio channels to stereo there:

<img width="1220" alt="image" src="https://github.com/jellyfin/jellyfin/assets/12995396/24584f59-af70-46c6-9158-9d2d7fa4a16c">

**Comment by gnattu** [comment]
Also, you may want to disable this: 
<img width="117" alt="image" src="https://github.com/jellyfin/jellyfin/assets/12995396/a2039b67-73a8-428f-b868-e4102917f7e6">

**Comment by dadastory** [comment]
> 对于JMP，我认为您需要转到客户端设置并在那里将音频通道更改为立体声：
> 
> <img alt="图像" width="1220" src="https://private-user-images.githubusercontent.com/12995396/337879830-24584f59-af70-46c6-9158-9d2d7fa4a16c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTc4NTYxOTEsIm5iZiI6MTcxNzg1NTg5MSwicGF0aCI6Ii8xMjk5NTM5Ni8zMzc4Nzk4MzAtMjQ1ODRmNTktYWY3MC00NmM2LTkxNTgtOWQyZDdmYTRhMTZjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjA4VDE0MTEzMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTgwMThlMzIxNGI4ZmJjMzQ5NzQzNzg5NDAwYmU2Yzc5MmUyZGE3YWY0Nzg1NDJmM2JkZDE4Y2Q2MzVmZjhkMTAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.SKPVJxaJZDZskyJFGnsT4vpb7uqTYWKBS3ZdtYqD3Kc">
![image](https://github.com/jellyfin/jellyfin/assets/84147842/05ef2ebf-57c6-44ec-8f32-2f7c8548122c)

The main thing is that the vocals are too quiet, and I had to turn the volume up to 100%, which is completely different from plotplayer playback

**Comment by TheHYPO35** [comment]
I thought I had noticed a week or two ago, around one the 10.9 updates that the volume on a series I was watching was quieter than it had been - I have to raise my PC volume much higher than normal to hear the show now. Did something change in terms of the volume level that should be produced under any of the 10.9 updates?

**Comment by macmini888** [comment]
I’ve also been noticing that I have to turn my sound system up much higher than before to get the volume to an acceptable place. This is on a Roku Ultra playing through a Denon 5.1 surround system playing both direct 5.1 Dolby Digital mixes and stereo AAC. 

I don’t really have a good memory for what baseline volume levels were before on Web or JMP otherwise I’d test that too.

**Comment by TheHYPO35** [comment]
> I’ve also been noticing that I have to turn my sound system up much higher than before to get the volume to an acceptable place. This is on a Roku Ultra playing through a Denon 5.1 surround system playing both direct Dolby Digital surround mixes and stereo AAC.
> 
> I don’t really have a good memory for what baseline volume levels were before on Web or JMP otherwise I’d test that too.

My experience is mainly in JMP where I can say for sure that I’ve never had to have my windows volume over 60 to hear shows, and all other system sound is overly loud (normally keep it around 40).

**Comment by macmini888** [comment]
Yes, my usual volume on the stereo is 50 from the Roku, but now I have to turn it up to 60 for Jellyfin.

**Comment by W-Floyd** [comment]
Also experiencing this, Jellyfin app on LG WebOS with a 5.1 sound bar. Some shows are extremely quiet and others are okay.

**Comment by Ironlenny** [comment]
I'm having the same issue. I can play the same file with VLC and it is obviously louder at the same system volume.

**Comment by nwbreneman** [comment]
> Also experiencing this, Jellyfin app on LG WebOS with a 5.1 sound bar. Some shows are extremely quiet and others are okay.

@W-Floyd I was having the same issue and finally found this: https://github.com/jellyfin/jellyfin-webos/issues/233

Seems to be a bug in the WebOS client. Scrolling up on the wheel of my remote finally fixed the low volume.

**Comment by W-Floyd** [comment]
@nwbreneman That fixed my issue, thanks so much!

**Comment by EionRobb** [comment]
As a workaround, adding `volume-gain=16` to the "Manual MPV Configuration" of the media player settings section works nicely.

**Comment by Mugane** [comment]
@EionRobb where is that exactly? admin or user settings? server file? etc? thanks

**Comment by EionRobb** [comment]
> @EionRobb where is that exactly? admin or user settings? server file? etc? thanks

User -> Client Settings -> Other
![image](https://github.com/user-attachments/assets/026a415f-fb49-4b79-b2ca-634ab8a16066)

`volume-gain=16` (as in 16dB) is a 4x volume boost - 2x would be `=8`
(Ignore the volume-max in the screenshot, I was playing around with trying to allow the volume to go above 100% like the mpv client can do)

**Comment by bassstorm** [comment]
Same issue here after upgrade to 10.9.x.

**Comment by bassstorm** [comment]
Oh, the client here is v1.11.1, just in case.
```
$ flatpak list | grep jellyfin
Jellyfin Media Player   com.github.iwalton3.jellyfin-media-player       1.11.1  stable  system
```

**Comment by rgroenewoudt** [comment]
It would be great if mpv's volume-gain is a UI setting.

**Comment by foegra** [comment]
well - would be good to enable only audio transcoding, even if video is direct playing. In my case if I force video transcoding (i have downmixing enabled on server as "night dialogue") - audio would become good as well. Jellyfin media player via Flatpak

![Image](https://github.com/user-attachments/assets/ab2bbc81-614e-41c7-98dc-4c44759b5b82)

**Comment by bassstorm** [comment]
Still an issue, devs please give this ticket more priority.

**Comment by manan-gup** [comment]
> Also, you may want to disable this: <img alt="image" width="117" src="https://private-user-images.githubusercontent.com/12995396/337879955-a2039b67-73a8-428f-b868-e4102917f7e6.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Njk0MTQzMTQsIm5iZiI6MTc2OTQxNDAxNCwicGF0aCI6Ii8xMjk5NTM5Ni8zMzc4Nzk5NTUtYTIwMzliNjctNzNhOC00MjhmLWI4NjgtZTQxMDI5MTdmN2U2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNjAxMjYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjYwMTI2VDA3NTMzNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWIyNWM2YWY2NThjMDliMDc4MzViY2YzMDY2OTI4ZDcyYjE5NjdlYTY2MWNmZDMxNjdjM2MxYTI1NDM4YzJlOGMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.-87y23w0nltMcYxnTc1mX7fw_Y1JCTcdEx6oQu948AE">

Disabling normalization fixed the issue for me.

**Comment by bassstorm** [comment]
> Disabling normalization fixed the issue for me.

More like a workaround to me. But thanks for sharing, disabling normalization deffo helps 🙏

**Comment by manan-gup** [comment]
Yes it's definitely only a workaround. There's probably a bug associated with that option.

**Comment by DamianPereira** [comment]
Would it be possible to support [replaygain](https://mpv.io/manual/stable/#options-replaygain) too? There's even album and track level gain, which could apply to episode/season too. This would obviously require files to have this metadata, which is not common at all, but could be analyzed with ffmpeg and set with some other tool.
